.bazelci/presubmit.yml

```diff
@@ -4,22 +4,48 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - yes | ./configure
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
+    - wget https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
     build_targets:
     - "//plaidml/keras:wheel"
+    build_options:
+    - "--config=linux_x86_64"
     test_targets:
     - "..."
+    test_options:
+    - "--cpu=linux_x86_64"
```

Comment:
To get the right set of options here, run `./configure` on a linux machine and see what file it produces (`.bazelrc.configure`). You can also take a peek on `buildkite-agent@lab-14:.bazelrc`.

It should only have this line: 
```
build --config=linux_x86_64
```

This would correspond to: 
```
build_options:
- --config=linux_x86_64
```

Reply:
Thanks for this explanation! I had tried to derive the test flags from the info messages from `bazelisk test //...` because I didn't know about using `.bazelrc.configure` to get the correct set of options.